en/micro-integrator/docs/references/connectors/salesforce-connectors/sf-rest-connector-config.md

```diff
@@ -383,6 +383,102 @@ To use the Salesforce REST connector, add the `<salesforcerest.init>` element in
 
 ---
 
```

Comment:
@Shakila - does this doc build properly?
Usually, the build fails when we have red highlighting.

Reply:
Yes, it's built successfully and page looks fine.
<img width="292" alt="SF" src="https://user-images.githubusercontent.com/2033965/102774484-1a08f880-43b1-11eb-97cd-4bc12e25607a.png">